modules/react/banner/lib/Banner.tsx

```diff
@@ -1,127 +1,101 @@
-  marginRight: space.xs,
-});
+const styles: CSSProperties = {
+  ...typeLevelsSubtextLarge,
+  cursor: 'pointer',
+  fontWeight: type.properties.fontWeights.medium,
+  textAlign: 'left',
+  transition: 'background-color 120ms',
+  '&:focus': {
+    outline: 'none',
+    ...focusRing({separation: 2}),
+  },
+  ...hideMouseFocus,
+};
 
-const BannerLabel = styled('div')({
-  flex: 1,
-});
+export const Banner = createComponent('button')({
+  displayName: 'Banner',
+  Component: ({children, model, ...props}: BannerProps, ref, Element) => {
+    const {hasError, isSticky, id, ...elemProps} = props;
+    const value = useDefaultModel(model, {hasError, isSticky, id}, useBannerModel);
 
-const BannerViewAll = styled('span')<BannerProps>(
-  {
-    textDecoration: 'underline',
-  },
-  ({variant}) => ({
-    display: variant === BannerVariant.Sticky ? 'none' : undefined,
-  })
-);
+    const palette = useThemedPalette(hasError ? 'error' : 'alert');
+    const themedBackgroundStyles: CSSProperties = {
+      '&:hover': {
+        backgroundColor: palette.hover,
+      },
+    };
 
-class Banner extends React.Component<BannerProps> {
-  static Variant = BannerVariant;
-  static ErrorType = ErrorType;
+    const isRTL = useIsRTL();
+    const borderBottomLeftRadius = isRTL ? 'borderBottomRightRadius' : 'borderBottomLeftRadius';
+    const borderTopLeftRadius = isRTL ? 'borderTopRightRadius' : 'borderTopLeftRadius';
+    const borderBottomRightRadius = isRTL ? 'borderBottomLeftRadius' : 'borderBottomRightRadius';
+    const borderTopRightRadius = isRTL ? 'borderTopLeftRadius' : 'borderTopRightRadius';
 
-  public render() {
-    const {
-      actionText = 'View All',
-      variant = BannerVariant.Full,
-      error = ErrorType.Alert,
-      label,
-      onClick,
-      ...props
-    } = this.props;
+    const borderStyleProps = {
+      [borderBottomLeftRadius]: borderRadius.m,
+      [borderTopLeftRadius]: borderRadius.m,
+      [borderBottomRightRadius]: isSticky ? 0 : borderRadius.m,
+      [borderTopRightRadius]: isSticky ? 0 : borderRadius.m,
+    };
 
-    const bannerIcon = error === ErrorType.Error ? exclamationCircleIcon : exclamationTriangleIcon;
-    const iconColor = error === ErrorType.Error ? colors.frenchVanilla100 : colors.blackPepper400;
-    const iconSize = 24;
+    const bannerProps = useBanner(value, elemProps, ref);
 
     return (
-      <BannerWrapper
-        aria-label={actionText + ': ' + label}
-        role="button"
-        variant={variant}
-        tabIndex={0}
-        onClick={onClick}
-        error={error}
-        {...props}
-      >
-        <BannerIcon icon={bannerIcon} color={iconColor} colorHover={iconColor} size={iconSize} />
-        <BannerLabel>{label}</BannerLabel>
-        <BannerViewAll variant={variant}>{actionText}</BannerViewAll>
-      </BannerWrapper>
+      <BannerModelContext.Provider value={value}>
+        <Flex
+          as={Element}
+          width={isSticky ? '222px' : '328px'}
```

Comment:
is there a chance someone wants to change this or is this per design spec? just curious!

Reply:
These are from the og component so assume they are spec. That said, since it is a style prop it should be easy to override if needed.